R/prepare_inter_sector_profile.R

```diff
@@ -8,6 +8,5 @@ prepare_inter_sector_profile <- function(sp_prod, europages_companies, ecoinvent
     left_join(ecoinvent_activities, by = "activity_uuid_product_uuid") |>
     left_join(ecoinvent_europages, by = c("country", "main_activity", "clustered", "activity_uuid_product_uuid")) |>
     left_join(isic, by = "isic_4digit") |>
-    add_avg_matching_certainty("completion") |>
-    exclude_rows("risk_category")
```

Comment:
Here is the main "bug". 
This line removes rows where `risk_category` is `NA` and therefore drops the rows we would want to output.